geonode/facets/tests.py

```diff
@@ -564,4 +564,119 @@ def test_thesauri_reloading(self):
         # Thesauri facets are cached.
         # Make sure that when Thesauri or ThesauriLabel change the facets cache is invalidated
         # TODO impl+test
+
         pass
+
+    def test_group_facet_api_call(self):
+        from django.contrib.auth.models import Group
+
+        group_admin = Group.objects.create(name="UserAdmin")
+        group_common = Group.objects.create(name="UserCommon")
+
+        group_profile_admin = GroupProfile.objects.create(group_id=group_admin, title="UserAdmin", slug="UserAdmin")
+        group_profile_common = GroupProfile.objects.create(group_id=group_common, title="UserCommon", slug="UserCommon")
```

Comment:
Lets' group the Group creation in a class function like the others `_create_resources`. Later you can just re-call them